app/controllers/content_pages_controller.rb

```diff
@@ -22,8 +22,8 @@ def new
 
   # GET /content_pages/1/edit
   def edit
-    doc = Govspeak::Document.new @content_page.markdown, sanitize: true, allowed_elements: ContentController::ALLOWED_TAGS
-    @md = doc.to_html
+    @md = MarkdownToHtml.new.translate_markdown(@content_page.markdown)
```

Comment:
instead of chaining the new method to translate_markdown like this, can you refactor the class so translate_markdown is a class method